bridge/contracts/libraries/errorCodes/BridgePoolErrorCodes.sol

```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: MIT-open-group
+pragma solidity ^0.8.11;
+
+library BridgePoolErrorCodes {
+    // BridgePool error codes
```

Comment:
Comment doesn't add anything. Consider [natspec](https://docs.soliditylang.org/en/develop/natspec-format.html) comments at the library level or for each error.